hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/BlockManagerImpl.java

```diff
@@ -117,6 +121,22 @@ public BlockManagerImpl(final Configuration conf,
             scm.getScmNodeManager(), scm.getEventQueue(), svcInterval,
             serviceTimeout, conf);
     safeModePrecheck = new SafeModePrecheck(conf);
+
+    long heartbeatInterval =  conf.getTimeDuration(
+        HddsConfigKeys.HDDS_HEARTBEAT_INTERVAL,
+        HddsConfigKeys.HDDS_COMMAND_STATUS_REPORT_INTERVAL,
```

Comment:
Should this HDDS_HEARTBEAT_INTERVAL_DEFAULT?

Reply:
Right.